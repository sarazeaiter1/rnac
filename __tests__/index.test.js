import All from '../';

test('Package exports correctly', () => {
  expect(All).toMatchSnapshot();
});
