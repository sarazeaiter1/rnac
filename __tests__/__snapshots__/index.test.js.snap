// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Package exports correctly 1`] = `
Object {
  "AudioRecorder": Object {
    "bindNativeEventListeners": [Function],
    "checkAuthorizationStatus": undefined,
    "destroy": [Function],
    "init": [Function],
    "onRecordingFinished": [Function],
    "onRecordingProgress": [Function],
    "pauseRecording": [Function],
    "removeListeners": [Function],
    "requestAuthorization": undefined,
    "startRecording": [Function],
    "stopRecording": [Function],
    "unbindNativeEventListeners": [Function],
  },
  "AudioUtils": Object {
    "CachesDirectoryPath": undefined,
    "DocumentDirectoryPath": undefined,
    "LibraryDirectoryPath": undefined,
    "MainBundlePath": undefined,
  },
  "Recorder": [Function],
  "TimeElapsed": [Function],
}
`;
